libp2p/protocols/rendezvous.nim

```diff
@@ -646,6 +655,10 @@ proc unsubscribeLocally*(rdv: RendezVous, ns: string) =
     return
 
 proc unsubscribe*(rdv: RendezVous, ns: string, peerIds: seq[PeerId]) {.async.} =
+  ## The async unsubscribe procedure removes peers from a namespace by
+  ## sending an "Unregister" message to each connected peer. The operation
+  ## is bounded by a timeout for unsubscribing from all peers.
```

Comment:
Where is this timeout?

Reply:
Here: https://github.com/vacp2p/nim-libp2p/pull/1203/files#diff-630b0a9abacaeb689dc2b124f4f34297daea166c589764b7276552b441047926R682